README.md

```diff
@@ -37,111 +37,86 @@ This will install a package named `xbb-tools` into your Conda environment. It sh
 
+## Running the Queries
 
-### Configuration for TCP over UCX
-
-To use UCX without NVLink, start the scheduler with the following:
+To run a query, starting from the repository root, go to the `queries` directory:
 
 ```bash
-LOGDIR="/raid/logs"
-
-DASK_UCX__CUDA_COPY=True DASK_UCX__TCP=True nohup dask-scheduler --interface ib0 --protocol ucx  > $LOGDIR/scheduler.log 2>&1 &
+cd tpcx_bb/queries/
 ```
 
-Start the workers with the following:
+Choose a query to run, and `cd` to that directory. We'll pick query 07.
 
 ```bash
-DEVICE_MEMORY_LIMIT="25GB"
-POOL_SIZE="30GB"
-LOGDIR="/raid/logs"
-WORKER_DIR="/raid/dask"
-MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
-
-
-dask-cuda-worker --rmm-pool-size=$POOL_SIZE --device-memory-limit $DEVICE_MEMORY_LIMIT --local-directory $WORKER_DIR  --rmm-pool-size=$POOL_SIZE --memory-limit=$MAX_SYSTEM_MEMORY --enable-tcp-over-ucx --scheduler-file lab-sched.json  >> $LOGDIR/worker.log 2>&1 &
+cd q07
 ```
 
-## Running the Queries
+Activate the conda environment you've created.
 
-To run the query, starting from the repository root, go to the `queries` directory:
+The queries assume that they can attach to a running Dask cluster. Command line arguments are used to determine the cluster and dataset on which to run the queries. The following is an example of running query 07.
 
 ```bash
-cd tpcx_bb/rapids-queries/
+python tpcx_bb_query_07.py --data_dir=$DATA_DIR --cluster_host=$SCHEDULER_IP --output_dir=$OUTPUT_DIR
 ```
 
-Choose a query to run, and `cd` to that directory. We'll pick query 07.
+- `data_dir` points to the location of the data
+- `cluster_host` corresponds to the schedyler address of the running Dask cluster
+    - In this case, this query would attempt to connect to a cluster running at `$SCHEDULER_IP`, which we configured during cluster startup
+- `output_dir` points to where the queries should write output files. If not specified, queries will write output into their respective directories
 
-```bash
-cd q07
-```
 
-Activate the conda environment with `conda activate rapids-tpcx-bb`.
+## Performance Tracking
 
-The queries assume that they can attach to a running Dask cluster. Command line arguments are used to determine the cluster and dataset on which to run the queries. The following is an example of running query 07.
+This repository contains support for performance-tracking automation using Google Sheets. If you install `gspread` and `oauth2client` in your conda environment, you will be able to pass `--sheet` and `--tab` arguments the query CLI to push performance metrics to Google Sheets. If you do not have these libraries installed, you will see `Please install gspread and oauth2client to use Google Sheets automation` after running a query. It's fine to ignore this output.
```

Comment:
```suggestion
`gspread` and `oauth2client` are included in the provided conda environment spec. Passing `--sheet` and `--tab` arguments during query execution will log query runtime in a Google Sheet. If you do not have these libraries installed, you will see `Please install gspread and oauth2client to use Google Sheets automation` after running a query. It's fine to ignore this output.
```

Reply:
Just realized these aren't actually in the conda environment file yet. Please add them.

Reply:
This is not currently true, but we can add them

Reply:
We’ll need to update the utilities code again if we add gspread as a dependency. Right now, the code will fail gracefully if gspread is not available. but, if it is, we will fail loudly if the google sheets configuration is incorrect.

It may be worth doing this in a follow-up

Reply:
Marking this as for a follow-up. Updated this section to address some of the edits.